Makefile

```diff
@@ -174,4 +174,4 @@ db-mongo-tunnel:
 
 db-postgres-tunnel:
 	@oc project $(TARGET_NAMESPACE)
-	@oc port-forward $(APP_NAME)-patroni-0 5432
+	@oc port-forward svc/$(APP_NAME)-patroni 5432
```

Comment:
What does adding this do?

For running reports on prod I usually deliberately point at a readonly replica, so want to make sure I understand this in case I need to do that in future :)

Reply:
@fwkendall , This will pointing to service and through service directly to leader pod. But 100% it will be pointing to RW replica pod. I do it because I want to update some record in dev/db to test certain things .... but if this is alarming for prod, I can create one more alias.

Reply:
Hmm the main thing is I'll need to update my instructions for the operations team on how to interact with prod.

Maybe two aliases makes sense - keep this one pointing to 0, so it provides more specificity, and create a new one db-postgres-service-tunnel with svc? For dev/test that's absolutely better in all case 🙌

Reply:
OK

Reply:
DONE